c2corg_ui/static/js/feed.js

```diff
@@ -114,14 +163,161 @@ app.FeedController = function(appAuthentication, appApi, appLang, imageUrl, ngeo
   this.isPersonal = !this.userId;
 
   /**
+   * @type {boolean}
+   * @export
+   */
+  this.showMobileBlock = /** @type {boolean} */ (JSON.parse(window.localStorage.getItem('showMobileBlock') || 'true'));
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.showAssoBlock = /** @type {boolean} */ (JSON.parse(window.localStorage.getItem('showAssoBlock') || 'true'));
+
+  /**
    * @type {ngeo.Location}
    * @public
    */
   this.ngeoLocation = ngeoLocation;
 
   this.getDocumentsFromFeed();
+
+  if (this.isPersonal) {
```

Comment:
This is not logical! It means for instance anonymous users won't see the forum feed...

Actually I think you cannot use the ``isPersonal`` flag. I have looked at the code and it is only used to decide if personal preferences (regions + activities) must be applied to the activity feed. See https://github.com/c2corg/v6_ui/blob/b0eb76c01c27d16c245b9c29644b75b0f769ca2c/c2corg_ui/static/js/apiservice.js#L407-L419

I really do think the forum block should not be managed in the feed directive (ditto for other blocks like the mobile app block) => it should be moved to its own directive which will be used in the homepage template.

Reply:
Haha, yeah i know but , he hasn't the good value, on profile, the var has false and on the feed the var has true. I dont know why but instead of break other thing that i didnt know i prefer to adapt the code this time.

Test it you will see :)

Reply:
I cannot test now, I have problems with my VPN access today, don't know why.

But I really do think your change is not logical, you only hacked the code to make it does what you want but with no logic. That makes the code confusing.

As I said, the forum has nothing to do with the activity feed and should be moved to a dedicated directive. What's the problem with this? It's much clearer.

Reply:
Concerning the blocks about the site intro and the mobile app, I also do think they should be moved out of the feed directive.

It would make sense to move https://github.com/c2corg/v6_ui/blob/4082be5f11d5b52098db206dd9cfbcd0e15623c2/c2corg_ui/static/partials/feed.html#L36-L82 to the home page template https://github.com/c2corg/v6_ui/blob/4082be5f11d5b52098db206dd9cfbcd0e15623c2/c2corg_ui/templates/index.html

Several reasons for this:
* the feed directive contains only feed related elements and functions, making it easier to read and maintain.
* partials are added to the JS built file. To make it as small as possible, it's better to move HTML code to templates whenever possible, so that people don't load it whereas they don't need it.
* it's more intuitive for developers to have this HTML code in the homepage template which describe the structure of the home page, eg. (existing) site intro, activity feed directive, forum feed directive and other information block such as the mobile app block or this extra site intro block.

I don't really understand the interest of the extra site intro block you have introduced. It's a duplicate of the existing one, isn't it? Then I recommend either to remove it or to drop the existing site intro block if we think the new one is more relevant and better placed in the right hand column.